experimental/generation/generator/packages/library/templates/standard/integerEntity.lu.lg

```diff
@@ -7,5 +7,5 @@
 - ```
 @ prebuilt number ${roles()}
 ${propertyName()}
-${operations('number', [1, 32, 99, 215, 1000])}
+${operations('number', [1, 32.4, 99, 215, 1000])}
```

Comment:
>32.4 [](start = 27, length = 4)

Did you mean to add a floating point number to this?  Are you eventually planning to handle telling the user it is a floating point number?

Reply:
I wrongly copy the content of numberEntity.lu.lg. Sorry about the mistake. Fix it.